pkgs/tools/security/masscan/default.nix

```diff
@@ -34,9 +34,10 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "Fast scan of the Internet";
-    homepage    = "https://github.com/robertdavidgraham/masscan";
-    license     = licenses.agpl3;
-    platforms   = platforms.unix;
+    homepage = "https://github.com/robertdavidgraham/masscan";
+    changelog = "https://github.com/robertdavidgraham/masscan/releases/tag/${version}";
+    license = licenses.agpl3;
```

Comment:
agpl3Plus or agpl3Only?

Reply:
Seems 3 only: https://raw.githubusercontent.com/robertdavidgraham/masscan/master/LICENSE

Reply:
Thanks for the feedback, guys.

Updated.